README.md

```diff
@@ -22,3 +23,53 @@ Where
 - If omitted, the link will apply to all products in the directory.
 - Test path is relative to the current directory, so will typically be just a filename.
 - Test result status is a status as defined in the wpt.fyi codebase.
+
+## How to contribute to wpt-metadata repository
+If you are a memeber of the web-platform-tests org, you can contribute to the wpt-metadata repo by sending out a PR directly or through a UI-based approach on wpt.fyi. We also have a Triage Metadata API available for trusted third parties
+
+### Triage through wpt.fyi UI
+For information about how to use the triage service on wpt.fyi, see here - UPCOMING
+
+### Triage Metadata API for trusted third parties
+In order to use the Triage Metadata API, first you need to log in on [wpt.fyi](https://wpt.fyi/) through the GitHub OAuth integration. For more information on wpt.fyi login, see [here](https://docs.google.com/document/d/1iRkaK6cGgXp3DKbNbPMVsYGMaOHO-5CfqEuLPUR_2HM)
+
+The logged-in user also needs to be a part of the web-platform-tests org. To join, please file an issue or contact us directly.
```

Comment:
```suggestion
The logged-in user also needs to belong to the ['web-platform-tests' GitHub organization](https://github.com/orgs/web-platform-tests/people). To join, please file an issue or contact us directly.
```
Where should they file an issue? (I suggest linking to the correct create URL, e.g. is it https://github.com/web-platform-tests/wpt/issues/new?)

How should they contact us directly? Is there an email address we should link them.

Reply:
Linked. I checked wpt.fyi and the wpt repository and looks like we don't give out email contacts. I will do the same here